projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -352,7 +357,7 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
     const formControlNames: (keyof FormInterface)[] = Object.keys(this.formControlNames) as (keyof FormInterface)[];
 
     const formValues: Observable<KeyValueForm>[] = formControlNames.map(key =>
-      this.formGroup.controls[key].valueChanges.pipe(
+      ((this.formGroup.controls[key] as unknown) as AbstractControl).valueChanges.pipe(
```

Comment:
Maybe

```suggestion
      ((this.formGroup.controls[key] as unknown) as TypedAbstractControl<FormInterface>).valueChanges.pipe(
```

?

Reply:
Hm, if we wanne type it here it would ahve to be:
```
((this.formGroup.controls[key] as unknown) as TypedAbstractControl<FormInterface[key]>).valueChanges.pipe(
```

But key is only available during runtime. Tbh since this is inside the base class I don't see any value add in trying to type it as the base class shouldn't be tied to specific types of control values.